### PR TITLE
Structural/Trilinos: Adding error in case of wrong convergence criteria is used

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

File: applications/StructuralMechanicsApplication/python_scripts/trilinos_convergence_criteria_factory.py [hunk 1]
```diff
@@ -49,5 +49,8 @@ def __init__(self, convergence_criterion_parameters):
             Residual = TrilinosApplication.TrilinosResidualCriteria(R_RT, R_AT)
             Residual.SetEchoLevel(echo_level)
             self.mechanical_convergence_criterion = TrilinosApplication.TrilinosOrCriteria(Residual, Displacement)
+            
+        else:
+            raise NameError("WARNING: Not convergence criterion available. The possibilities are: \ndisplacement_criterion \nresidual_criterion \nnd_criterion \nor_criterion")
```
Comment (rubenzorrilla) [inline applications/StructuralMechanicsApplication/python_scripts/trilinos_convergence_criteria_factory.py]: I'd change the message error. This is not a warning. Besides there is a typo (change \nnd_criterion by \nand_criterion).

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]

File: applications/StructuralMechanicsApplication/python_scripts/trilinos_convergence_criteria_factory.py (hunk 1 above)
Comment (philbucher) [inline applications/StructuralMechanicsApplication/python_scripts/trilinos_convergence_criteria_factory.py]: can you please make it consistent with the [non-trilinos version](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/convergence_criteria_factory.py#L58)?

**Review by philbucher — CHANGES_REQUESTED** [review]

**Comment by loumalouomega** [comment]
@philbucher Ok

**Review by philbucher — APPROVED** [review]
thx

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**